docs/deploy.rst

```diff
@@ -0,0 +1,228 @@
+
+.. code-block:: console
+
+    sh$ croud organizations create --name YourOrganizationName
+
+Verify that you are a member of the organization you created by giving the
+following command:
+
+.. code-block:: console
+
+    sh$ croud organizations list
+    +--------------------------------------+----------------+-------------+
+    | id                                   | name           |   plan_type |
+    |--------------------------------------+----------------+-------------|
+    | 59c7961b-33a2-4478-9e98-2a6b3f54f60e | MyOrganization |           1 |
+    +--------------------------------------+----------------+-------------+
+
+You will get a list of organizations that hold you as a member. The list shows
+the org-ID, name, and plan type of each shown organization.
+
+
+Create a project
+================
+
+Each project you create is tied to your default region. Thus also every CrateDB
+product that you deploy in a project.
+
+.. CAUTION::
+   To interact with a particular product, make sure to run the command in the
+   right region by passing the ``--region`` parameter as well as the
+   corresponding option to the command.
+
+The ``region`` setting determines in which geographical location your data is
+stored. It also specifies which CrateDB products, such as clusters, you will
+find available.
+
+You can set the following regions:
+
++-------------------+---------------------+
+| Region            | Command-line option |
++===================+=====================+
+| Azure East-US     | eastus.azure        |
++-------------------+---------------------+
+| Azure East-US     | eastus2.azure       |
++-------------------+---------------------+
+| Azure West-Europe | westeurope.azure    |
++-------------------+---------------------+
+| Bregenz           | bregenz.a1          |
++-------------------+---------------------+
+
+This tutorial uses the region ``bregenz.a1``.
+
+To set the region run the following command:
+
+  .. code-block:: console
+
+     sh$ croud config set --region bregenz.a1
+     ==> Info: Region switched to bregenz.a1
+
+Once you have set the region, all Croud commands will be executed for
+that region.
+
+Use the following command to create a project in your new organization and
+specify a project name:
+
+.. code-block:: console
+
+    sh$ croud projects create --name YourProjectName
+
+You have added a project to your organization.
+
+.. SEEALSO::
+
+  Look up the `Croud CLI Reference`_ for help using the ``croud organizations``
+  or ``croud projects`` commands.
+
+
+Review available products
+=========================
+
+Now that you have set a default region, created an organization and project, you
+can call up a list of products that are available in your region.
+
+Run the following command:
+
+.. code-block:: console
+
+   sh$ croud products list --region bregenz.a1
+   +-------------------------------------------------------------+----------+-------------------+---------------------------+--------+
```

Comment:
I don't want to include this table in the getting started docs. it should be in the reference

we can allude to its contents, and link through to the full table elsewhere